swatch-picker/open-swatch-picker.js

```diff
@@ -0,0 +1,2204 @@
+
+
+const styles = new CSSStyleSheet()
+styles.replaceSync(`
+    dialog {
+        border-radius: 1rem;
+
+        &::backdrop {
+            background: rgba(0, 0, 0, 0.5);
+            opacity: 0.75;
+        }
+
+        & div[scale] {
+            display: flex;
+
+            & h2 {
+                display: none;
+            }
+            & h2[active] {
+                grid-column: span 13;
+                display: block;
+            }
+            & div[swatch] {
+                margin: 2px;
+
+                & button {
+                    width: 2.5rem;
+                    height: 2.5rem;
+                    border: none;
+                    border-radius: 0.5rem;
+                    cursor: pointer;
+                    outline: 2px solid transparent;
+
+                    &:hover {
+                        outline: 2px solid;
+                    }
+                }
+                & span {
+                    display: none;
+                }
+            }
+        }
+    }
+    [part="swatch-picker"] {
+        display: flex;
+        align-items: center;
+        gap: 1rem;
+    }
+`);
+const template = document.createElement("template")
+template.innerHTML = `
```

Comment:
```suggestion
template.innerHTML = /* html */`
```

This should make this code syntax highlight as HTML in some editors.